docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -1,11 +1,10 @@
 # Healthcare and Life Science Blueprint
 
 > [!NOTE]
-> This document uses SlurmGCP v5 version of hcls blueprint. If you want to
-> use SlurmGCP v6 version, please refer to this [blueprint](../examples/hcls-blueprint.yaml).
+> This document uses SlurmGCP v6 version of hcls blueprint. If you want to
+> use SlurmGCP v5 version, please refer to this [blueprint](./hcls-blueprint-v5-legacy.yaml).
```

Comment:
`/ghpc create docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml` below needs to be updated as v6 blueprint no longer lives in this folder.

Reply:
Updated to point to v6 example.